content/ghent2025/guide.md

```diff
@@ -0,0 +1,229 @@
+    One reasonable price to get unlimited local transportation and entrance to
+    attractions, monuments, museums, and other sights. The local city bus is fairly
+    extensive and will take you even to other nearby cities, like Bruges.
+
+1. **The venue: *HoGhent SchoonMeersen***
+
+    1. **Finding your way around.**
+
+        Take the tram in from city center to the HoGhent SchoonMeersen campus. Don’t
+        expect to see a lot of signage directing you though. This is a volunteer run
+        conference without big marketing budgets. There won’t be a lot of students
+        around at this time, so if you see groups of people carrying laptops, they’re
+        probably a good bet to follow. This [Google maps link](https://maps.app.goo.gl/STnKypnvg1btZP947)
+        will take you directly to the front door of the conference lobby, where
+        you’ll see vendor tables, coffee, pastries, etc.
+
+        The conference is primarily in the *B* building. The main bathrooms are on the
+        ground floor, through the large doors to the left. Other bathrooms will
+        alternate genders on each of the floors. Several conference rooms are also on
+        the ground floor. Many are on upper floors, you’ll find the elevator and stairs
+        to the right. The rooms are coded as `<Building>.<Floor>.<Room>`. In
+        other words, *B.3.039* is building *B*, third floor, room *39*.
+
+    1. **Paying attention to the schedule and following updates**
+
+        Wifi access will be shared during the conference introduction in the *D auditorium*.
+        You’ll want this for the schedule and other such, but please don’t abuse it.
+
+        [Open the schedule](https://cfp.cfgmgmtcamp.org/2025/schedule/) on your phone
+        and/or laptop. Plan ahead which talks, or tracks, or speakers you’re excited to
+        see and figure out where the rooms are. Then pay attention around you. If
+        schedule changes are required, such as running 15 minutes behind or the like,
+        they won’t be announced over a loudspeaker. Instead, you’ll overhear it or see
+        crowds of people start moving. Don’t hesitate to ask someone what’s going on.
+        We’re all helping each other out.
+
+    1. **The common area and hallway track**
+
+        The lobby is where vendors will have their tables set up. There will be coffee,
+        fruits, pastries, and such. And there will be a lot of people excited to talk to
+        each other. This is where most of the hallway track happens. When you’re done
+        with a talk and you don’t have anything in particular to do, you should come
+        here. If you are in the lobby and it seems particularly empty, then there’s
+        probably a talk happening. If you didn’t expect that, ask someone. There may
+        have been a schedule change that didn’t get published.
+
+    1. **Swag pickup**
+
+        Walter will have to do this section 😀
```

Comment:
@walterheck this is for you

Reply:
Pickup on Monday before the keynotes in the hall where the sponsors are. Foe questions email me at walterheck@helixiora.com